### PR TITLE
Adding a new plugin which just disables the LEDs until a button press

### PR DESCRIPTION
**Pull request by oxivanisher**
This way, we can keep the raspberry pi running and just switch off the leds until a button is pressed.

### CODE REVIEW

**Comment by Tihmann** [comment]
Add oxivanisher's new plugin

File: wordclock_plugins/leds_off/plugin.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        '''
+        emptied = False
+        while True:
+            """Draw nothing."""
+            if not emptied:
+                for i in range(114):
+                    wcd.setPixelColor(i, Color(0, 0, 0))
+                wcd.show()
+                emptied = True
+            if not GPIO.input(22) or GPIO.input(8):
```
Comment (bk1285) [inline wordclock_plugins/leds_off/plugin.py]: Currently, the wordclock layout/concept undergoes some changes. Afterwards, I'll merge it back, changing this part to a more generic one (GPIO 22 or 8 is too specific and might not be valid for all wordclock setups)

**Comment by oxivanisher** [comment]
Is this still ongoing?

**Comment by oxivanisher** [comment]
Can I update my code to implement this plugin, I still would like to use it. :)

**Comment by bk1285** [comment]
Hi @oxivanisher 

just adopted your work to the latest changes and merged into develop: https://github.com/bk1285/rpi_wordclock/tree/develop

Best,
Bernd

**Pull request closed without merging**